high_card.rb

```diff
@@ -3,3 +3,39 @@
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+
+ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+suits = [ "hearts", "spades", "clubs", "diamonds" ]
+
+deck=[]
+
+ranks.each do |rank|
+  suits.each do |suit|
+    deck.push([rank,suit])
+  end
+end
+
+deck.shuffle!
+
+players = []
+
++puts players.length.to_s = "{n} players so far. Enter a player name, or type 'play':"
++name = gets.chomp.to_s
++if name === "play"
```

Comment:
I think here, the idea is to continue to name players until `play` is typed in. Maybe a loop would be the right fit here.